nextcord/ext/commands/bot.py

```diff
@@ -653,7 +653,12 @@ def _call_module_finalizers(self, lib: types.ModuleType, key: str) -> None:
                 if _is_submodule(name, module):
                     del sys.modules[module]
 
-    def _load_from_module_spec(self, spec: importlib.machinery.ModuleSpec, key: str) -> None:
+    def _load_from_module_spec(
+            self,
+            spec: importlib.machinery.ModuleSpec,
+            key: str,
+            extras: Optional[Dict[str, Any]] = None
```

Comment:
Maybe use Missing instead?

Reply:
Based on epics thumbs down imma resolve this